release/release.md

```diff
@@ -0,0 +1,112 @@
+# How to do a Kata Containers Release
+
+Hi if you are reading this document you may also want to create a kata Containers Release.
+
+The Kata Containers Release Process is defined in the follwoing documents.
+
+https://github.com/kata-containers/documentation/blob/master/Releases.md
+
+To simply the process of read each release we have created a checklist for it.
+
+https://github.com/kata-containers/documentation/blob/master/Release-Checklist.md
+
+In order to simplify the process of do the Release Checklist we have semi-automated most of the process.
```

Comment:
s/In order to simplify the process of do the Release Checklist/To simplify the Release Checklist process/